AndroidAssignment/app/src/main/java/org/sopt/di/DatabaseModule.kt

```diff
@@ -26,5 +30,33 @@ object DatabaseModule {
 
     @Provides
     fun provideUserDao(database: UserDatabase): UserDao =
-        database.userDao
+            database.userDao
+
+    @Provides
+    @Singleton
+    fun provideProfileDatabase(@ApplicationContext context: Context): ProfileDatabase =
+            Room.databaseBuilder(
+                    context.applicationContext,
+                    ProfileDatabase::class.java,
+                    "profile database"
+            ).fallbackToDestructiveMigration()
+                    .build()
```

Comment:
```suggestion
            Room.databaseBuilder(
                    context,
                    ProfileDatabase::class.java,
                    "profile database"
            ).fallbackToDestructiveMigration()
                    .build()
```

application context를 주입받는 데 굳이 applicationContext를 또 가져오는 건 어색한 것 같음.

Reply:
잉 나 왜저랬지..ㅋㅋㅋㅋㅋㅋ... ㄹㅇ 도라인가벼.. 저건 바로 수정한다